pkg/controller/restore/restore_controller.go

```diff
@@ -99,7 +99,7 @@ func (c *Controller) processNextWorkItem() bool {
 		} else if perrors.Find(err, controller.IsIgnoreError) != nil {
 			klog.V(4).Infof("Restore: %v, ignore err: %v", key.(string), err)
 		} else {
-			utilruntime.HandleError(fmt.Errorf("Restore: %v, sync failed, err: %v, requeuing", key.(string), err))
+			utilruntime.HandleError(fmt.Errorf("restore: %v, sync failed, err: %v, requeuing", key.(string), err))
```

Comment:
🤔

Reply:
This is to try a lint for https://github.com/golang/go/wiki/CodeReviewComments#error-strings. But it isn't introduced by this PR. So I restore to the previous.